Cat2Cat/main.m

```diff
@@ -15,6 +15,8 @@
 
 static VICatalogWalkerParameters *parametersFromLegacyArguments(int argc, const char * argv[]);
 static VICatalogWalkerParameters *parametersFromArguments(int argc, const char * argv[]);
+static BOOL validateMethodNamePrefix(NSString *methodNamePrefix, NSError **error);
+static void exitWithLaunchArgAndError(const char *launchArg, NSError *error);
```

Comment:
It might be clearer to call it `exitWithCommandNameAndError(const char *commandName, …`, since `argv[0]` is the command name.